From 0b2fb80d9fa5be351a6bff3662cf27e50c940852 Mon Sep 17 00:00:00 2001
From: Jose-Luis Rivas <me@ghostbar.co>
Date: Fri, 1 Apr 2016 20:33:28 -0400
Subject: [PATCH] fix path to llvm-cmake-path

---
 cmake/modules/SwiftSharedCMakeConfig.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/modules/SwiftSharedCMakeConfig.cmake b/cmake/modules/SwiftSharedCMakeConfig.cmake
index f9ed4f6..f8fad5d 100644
--- a/cmake/modules/SwiftSharedCMakeConfig.cmake
+++ b/cmake/modules/SwiftSharedCMakeConfig.cmake
@@ -147,7 +147,7 @@ macro(swift_common_standalone_build_config product is_cross_compiling)
   find_program(LLVM_TABLEGEN_EXE "llvm-tblgen" "${${product}_NATIVE_LLVM_TOOLS_PATH}"
     NO_DEFAULT_PATH)
 
-  set(LLVM_CMAKE_PATH "${LLVM_BINARY_DIR}/share/llvm/cmake")
+  set(LLVM_CMAKE_PATH "/usr/share/llvm-3.6/cmake")
   list(APPEND CMAKE_MODULE_PATH "${LLVM_CMAKE_PATH}")
 
   set(LLVMCONFIG_FILE "${LLVM_CMAKE_PATH}/LLVMConfig.cmake")
-- 
2.8.0.rc3

